oauth2_http/javatests/com/google/auth/oauth2/UserAuthorizerTest.java

```diff
@@ -376,7 +443,7 @@ public void getAndStoreCredentialsFromCode_getAndStoresCredentials() throws IOEx
     MockTokenServerTransportFactory transportFactory = new MockTokenServerTransportFactory();
     transportFactory.transport.addClient(CLIENT_ID_VALUE, CLIENT_SECRET);
     transportFactory.transport.addAuthorizationCode(
-        CODE, REFRESH_TOKEN, accessTokenValue1, GRANTED_SCOPES_STRING);
+        CODE, REFRESH_TOKEN, accessTokenValue1, GRANTED_SCOPES_STRING, null);
```

Comment:
There is no method signature without additional params now. Since it is testing method I did not overload it. So adding null for places where additional params are not used.

Reply:
ah, it's a mock method.